### PR TITLE
Add Pablo to the projects list

### PR DESCRIPTION
**Pull request by ajsaraujo**
I created this small Angular project. The app is pretty straightforward and I left some issues open. I think anyone can contribute to it and thus it's perfect for Up For Grabs.

### CODE REVIEW

File: _data/projects/pablo.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Pablo
+desc: A Color Palette management app
+site: https://ajsaraujo.github.io/pablo/index.html
```
Comment (ritwik12) [inline _data/projects/pablo.yml]: Good for users, but how will someone trying to contribute will reach to the Github repo from there?

File: _data/projects/pablo.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+tags:
+  - angular
+  - website
+  - typescript
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ajsaraujo/pablo/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/pablo.yml]:
```suggestion
  link: https://github.com/ajsaraujo/pablo/labels/good%20first%20issue
```
This would be better.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/pablo.yml (hunk 1 above)
Comment (ajsaraujo) [reply _data/projects/pablo.yml]: Makes sense, I didn't thought of that.

**Comment by ajsaraujo** [comment]
Thanks for the input. The requested changes were made and are now waiting for review.

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/pablo.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/pablo.yml]: Cool

**Comment by ritwik12** [comment]
@ajsaraujo Thanks!!

**Pull request merged**